samigo/samigo-api/src/java/org/sakaiproject/tool/assessment/data/ifc/assessment/ItemDataIfc.java

```diff
@@ -36,6 +36,12 @@ public interface ItemDataIfc extends Comparable<ItemDataIfc>, java.io.Serializab
   public static final Integer ACTIVE_STATUS = 1;
   public static final Integer INACTIVE_STATUS = 0;
 
+  public static final int ITEM_NOT_CANCELED = 0;
```

Comment:
Shall we use Integer instead of a basic int?

Reply:
The switch statements don't like `Integer`'s :thinking:

Reply:
Yeah but we can use an enum, no problem, It's fine, thanks Markus!